gymfc.ini

```diff
@@ -3,7 +3,7 @@
 # In most cases this should not be modified
 [DEFAULT]
 
-SetupFile = /home/gymfc/local/share/gazebo/setup.sh
```

Comment:
I'm in the process of submitting a PR to be able to parse environment variables, please change this to `$HOME/local/share/gazebo/setup.sh`.

Reply:
Please see this PR, https://github.com/wil3/gymfc/pull/44